packages/app/src/resource/BotEditor.tsx

```diff
@@ -75,20 +71,18 @@ export function BotEditor(): JSX.Element | null {
       try {
         const code = await getCode();
         const sourceCode = await medplum.createAttachment(code, 'index.ts', 'text/typescript');
-        const operations: PatchOperation[] = [];
-        if (bot?.sourceCode) {
-          operations.push({
-            op: 'replace',
+        const operations: PatchOperation[] = [
+          {
+            op: 'add',
             path: '/sourceCode',
             value: sourceCode,
-          });
-        } else {
-          operations.push({
+          },
+          {
             op: 'add',
-            path: '/sourceCode',
+            path: '/executableCode',
```

Comment:
This won't work, will it?  `executableCode` has to be CJS, it won't work with TypeScript?

Reply:
I guess I don't understand how it is currently working? Since all I did was set the `executableCode` to what gets passed into the `$deploy` operation previously

Reply:
Ah I see, `getCodeOutput` is compiling the code...